pkg/operator/etcdendpointscontroller/etcdendpointscontroller.go

```diff
@@ -112,7 +113,7 @@ func (c *EtcdEndpointsController) syncConfigMap(recorder events.Recorder) error
 			}
 		}
 	} else if !errors.IsNotFound(err) {
-		return fmt.Errorf("couldn't get configmap %s/%s: %w", operatorclient.TargetNamespace, "etcd-endpoints", err)
+		klog.Warningf("couldn't get configmap %s/%s: %w", operatorclient.TargetNamespace, "etcd-endpoints", err)
```

Comment:
Maybe indicate that it will be re-created so that it will be obvious what the consequence of the warning is (i.e. an issue that will be automatically recovered)?

Reply:
makes senes

Reply:
updated